allensdk/brain_observatory/behavior/behavior_project_cache/project_apis/data_io/behavior_neuropixels_project_cloud_api.py

```diff
@@ -146,9 +128,55 @@ def f():
             str(session_data_path), probe_meta=probe_meta
         )
 
+    @staticmethod
+    def _return_one_session_metadata(input_table: pd.DataFrame,
```

Comment:
this is not specific to behavior neuropixels. Can you please make this more generic such as `get_one_row` and place in `dataframe_utils` ?